images/autobumper/autobump.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET_DIR=$(mktemp -d -t ci-XXXXXXXXXX)
+readonly TARGET_DIR
+
+echo "cloning ${origin} to ${TARGET_DIR} ..."
```

Comment:
Why do we need this in the image? For caching it?

Reply:
Done.